prov/mlx/src/mlx_av.c

```diff
@@ -116,9 +162,14 @@ static int mlx_av_insert(
 		} else {
 			if (av->eq) {
 				mlx_av_write_event( av, i,
-						MLX_TRANSLATE_ERRCODE(status),
-						context);
+					MLX_TRANSLATE_ERRCODE(status),
+					context);
 			}
+			break;
```

Comment:
If we take this path, we leak ep_params.address

Reply:
Now free calls before break. Problem should go away.